### PR TITLE
[Traction] Update helm values for production and test environments

### PR DESCRIPTION
**Pull request by i5okie**

### CODE REVIEW

File: helm-values/traction/values-production.yaml [hunk 1]
```diff
@@ -1,15 +1,51 @@
 ingressSuffix: -prod.apps.silver.devops.gov.bc.ca
 acapy:
+  ledgers.yml:
+    - id: candy-prod
+      is_production: true
+      is_write: true
+      genesis_url: 'http://test.bcovrin.vonx.io/genesis'
```
Comment (loneil) [inline helm-values/traction/values-production.yaml:7]: Question for @esune what would the real Candy Prod ledger genesis url be?

File: helm-values/traction/values-production.yaml [hunk 2]
```diff
@@ -21,10 +57,39 @@ tenant_proxy:
...
     targetMemoryUtilizationPercentage: 80
+ui:
+  showOIDCReservationLogin: true
+  oidc:
+    active: true
+    showInnkeeperAdminLogin: true
+    authority: https://loginproxy.gov.bc.ca/auth/realms/digitaltrust-nrm
+    jwksUri: https://loginproxy.gov.bc.ca/auth/realms/digitaltrust-nrm/protocol/openid-connect/certs
+  ariesDetails:
+    ledgerDescription: "bcovrin-test"
```
Comment (loneil) [inline helm-values/traction/values-production.yaml:72]: Can set this to `candy-prod` so it displays in the UI but I actually need to revisit this field now that it's multi-ledger (doing that in my current Traction issue)

File: helm-values/traction/values-production.yaml [hunk 3]
```diff
@@ -1,15 +1,51 @@
...
+        auto_approve: false
+        auto_issuer: false
+  walletStorageConfig:
+    url: traction-database-primary:5432
+  walletStorageCredentials:
+    admin_account: walletman
+    existingSecret: traction-database-acapy
+    secretKeys:
+      adminPasswordKey: walletman-password
+      userPasswordKey: acapy-password
```
Comment (loneil) [inline helm-values/traction/values-production.yaml:31]: I'm not actually too clear on these fields but are they meant to be in the prod one and not the test one?

**Review by loneil — COMMENTED** [review]
Couple tweaks/questions

File: helm-values/traction/values-production.yaml (hunk 1 above)
Comment (esune) [reply helm-values/traction/values-production.yaml:7]: See here: https://github.com/bcgov/trust-over-ip-configurations/blob/284153d343e84e23997286672984ac5b74b542a3/openshift/templates/issuer-agent/config/prod/ledgers-candy.yaml#L4

File: helm-values/traction/values-production.yaml (hunk 3 above)
Comment (esune) [reply helm-values/traction/values-production.yaml:31]: Prod is already using CrunchyDB, test needs to be updated so won't have this set quite yet. This coupled with disabling the postgres component is required to use Crunchy - c.c.: @i5okie

Comment (loneil) [reply helm-values/traction/values-production.yaml:31]: Ah ok. I see

File: helm-values/traction/values-production.yaml (hunk 1 above)
Comment (esune) [reply helm-values/traction/values-production.yaml:7]: This needs to be updated to the actual value - currently fixed manually in `prod` temporarily

**Review by esune — CHANGES_REQUESTED** [review]

File: helm-values/traction/values-production.yaml [hunk 4]
```diff
@@ -1,15 +1,51 @@
...
+  plugin-config.yml:
+    traction_innkeeper:
+      innkeeper_wallet:
+        print_key: false
+        print_token: false
+        connect_to_endorser:
+          - endorser_alias: candy-prod-endorser
+            ledger_id: candy-prod
+        create_public_did:
+          - candy-prod-endorser
```
Comment (loneil) [inline helm-values/traction/values-production.yaml:19]: This should be `candy-prod` instead

File: helm-values/traction/values-production.yaml (hunk 1 above)
Comment (i5okie) [reply helm-values/traction/values-production.yaml]: updated

File: helm-values/traction/values-production.yaml (hunk 2 above)
Comment (i5okie) [reply helm-values/traction/values-production.yaml]: updated

File: helm-values/traction/values-production.yaml (hunk 4 above)
Comment (i5okie) [reply helm-values/traction/values-production.yaml]: updated

**Review by loneil — APPROVED** [review]

**Review by esune — DISMISSED** [review]

**Review by esune — APPROVED** [review]

**Pull request merged**